webextensions/api/devtools.json

```
@@ -74,6 +91,90 @@
                     "version_added": false
                   }
                 }
+              },
+              "contextSecurityOrigin": {
+                "__compat": {
+                  "support": {
+                    "chrome": {
+                      "version_added": true
+                    },
+                    "edge": "mirror",
+                    "firefox": {
+                      "version_added": false
+                    },
+                    "firefox_android": "mirror",
```

Even if support for the API is ever added, it won't be on Android because the devtools API is unsupported on Android. So replace mirroring with marking as unsupported, like this:

```suggestion
                    "firefox_android": {
                      "version_added": false
                    },
```

---

@Rob--W I'm not sure how much of a problem this will be because the Linter calls this an "error" and says you should use "mirror". I've made the change, lets see what happens.